utils/validations/test_mocks.go

```diff
@@ -25,7 +27,7 @@ const (
 )
 
 var (
-	versionApiUrl = "/%s/api/v1/system/version"
+	versionApiUrl = "/%s/%ssystem/version"
```

Comment:
I think you're missing '/' after the second %s

Reply:
No, you can see the logic and usage
![image](https://github.com/user-attachments/assets/4f255954-e9a1-4217-a7c7-50a9c8c6daa0)